src/__forks__/Map.native.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
```

Comment:
Eliminated `@providesModule` so that we can kill this: https://github.com/facebook/react-native/blob/master/packager/blacklist.js#L21